packages/react-dev-overlay/src/internal/components/Overlay/maintain--tab-focus.ts

```diff
@@ -804,11 +804,17 @@ var focusSummary = {
 }
 
 function makeFocusableForeignObject() {
-  var fragment = document.createElement('div')
-  fragment.innerHTML =
-    '<svg><foreignObject width="30" height="30">\n      <input type="text"/>\n  </foreignObject></svg>'
+  // Returns <foreignObject width="30" height="30">\n      <input type="text"/>\n  </foreignObject>
```

Comment:
Nit: Clean this comment up and add more context. E.g.

// Construct &lt;foreignObject width="30" height="30">&lt;input type="text"/>&lt;/foreignObject>
// in a way that does not raise a Trusted Types violation.